### PR TITLE
Fix plates 1 and 2 IC + rerun

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
I fixed the CellProfiler pipeline for illumination correction in this PR and reran these plates through CellProfiler analysis.

I reran the code to run through the processing of the outputted SQLite files. I noticed that VSCode determined that Plates 3 and 3 prime are different for normalization and feature selection. @MattsonCam, Can you please confirm that these reruns for plates 3 and 3 prime did not change the values?

### CODE REVIEW

**Comment by MattsonCam** [comment]
It looks like the dataframes are identical for both plates 3 and 3' across all datasets

File: 2.cellprofiler_analysis/scripts/nf1_analysis.py [hunk 1]
```diff
@@ -86,3 +88,4 @@
         sqlite_name=plate,
```
Comment (MattsonCam) [inline 2.cellprofiler_analysis/scripts/nf1_analysis.py:88]:
```
# run through each plate with each set of paths based on dictionary
for plate, info in plates_info_dictionary.items():
    path_to_pipeline = info["path_to_pipeline"]
    path_to_images = info["path_to_images"]
    print(f"Running analysis on {plate}!")
    # run analysis pipeline
    cp_utils.run_cellprofiler(
        path_to_pipeline=path_to_pipeline,
        path_to_output=path_to_output,
        path_to_images=path_to_images,
        # name each SQLite file after plate name
```

May also consider adding the `path_to_output.mkdir(exist_ok=True)` here like you did in the 1.cellprofiler_ic/nf1_ic.ipynb file

File: 3.processing_features/processing_features.sh [hunk 2]
```diff
@@ -9,4 +9,7 @@ conda activate nf1_cellpainting_data
 jupyter nbconvert --to python --output-dir=scripts/ *.ipynb
 
 # run the python scripts in order (from convert+merge, annotate, normalize, and feature select)
-python scripts/merge_sc_cytotable.py && python scripts/annotate_sc.py && python scripts/normalize_sc.py && python scripts/feature_select_sc.py
+python scripts/0.merge_sc_cytotable.py
```
Comment (MattsonCam) [inline 3.processing_features/processing_features.sh:12]: Good job, this does look cleaner

**Review by MattsonCam — APPROVED** [review]
Good job overall! This was a smaller pr than I expected. I made a few comments, but if you want feedback on anything else specifically, just let me know. I'll tag you just in case @jenna-tomkinson .

File: 2.cellprofiler_analysis/scripts/nf1_analysis.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 2.cellprofiler_analysis/scripts/nf1_analysis.py:88]: I agree, so I added this in the code cell above instead and edited the documentation.

**Comment by jenna-tomkinson** [comment]
@MattsonCam Thank you for the review! I will be merging now!

**Pull request merged**